### ISSUE TITLE
Machine account for releases 

### ISSUE THREAD

**Issue by ollien** [vscode-neovim/vscode-neovim#2036]
Github's terms of service allow for machine accounts, which are accounts otherwise dedicated for automated actions. Might we consider making one for this project and using its PAT for release-please? This would free up @theol0403's account, and make it more clear when he's the one doing the contributing! 

I imagine we'd want only select trusted maintainers to have access to this account.

**Comment by theol0403** [comment]
No objections. The reason it's me currently, is that using a bot token doesn't trigger CI when it makes a PR. But a machine account would be fine.

**Comment by ollien** [comment]
Cool. I think you'd have to do this, since it requires editing repository secrets :)

**Comment by xiyaowong** [comment]
Use release-please app?

Using the GitHub app also makes it convenient to use tags for release-please operations.
https://github.com/googleapis/release-please#step-3-rerun-release-please

**Comment by ollien** [comment]
Would the app have to be hosted somewhere? That's how the README for that app looks to me, but I've not used this feature before.

**Comment by theol0403** [comment]
@xiyaowong the problem with your solution is that CI is not triggered for the release PRs

**Comment by xiyaowong** [comment]
You mean
![image](https://github.com/vscode-neovim/vscode-neovim/assets/47070852/858cc32c-0344-4cc7-b0d0-e8e49a188eec)
?

This doesn't matter, does it?

**Comment by xiyaowong** [comment]
???

**Comment by xiyaowong** [comment]
Does it make any sense for our project to test the release PR?

**Comment by ollien** [comment]
Probably, yes. It's a nice check to ensure that tests pass for what we're about to release, even if they run on master.

**Comment by xiyaowong** [comment]
Release-please only updates the changelog. 😅

**Comment by ollien** [comment]
Yeah, but it's also the gate for our release. It's generally a good idea and good practice to make it as hard as possible to publish a release with broken tests.

**Comment by theol0403** [comment]
There is a small chance that merging multiple PRs at the same time (without rebasing first) can cause test failures that aren't detected in the separate PRs.

**Comment by xiyaowong** [comment]
Too tangled up! Then let's create a new bot account.
